lms/tasks/event.py

```diff
@@ -0,0 +1,12 @@
+from lms.events.event import BaseEvent
+from lms.tasks.celery import app
+
+
+@app.task
+def insert_event(event: dict) -> None:
+    with app.request_context() as request:  # pylint: disable=no-member
+        with request.tm:
+            # pylint:disable=import-outside-toplevel,cyclic-import
```

Comment:
Needed because of the circular dependency between this and the service. I reckon that's the right design in any case.